src/app/brukerprofil/adresse/gateadresse/GateadresseForm.tsx

```diff
@@ -83,7 +83,12 @@ function GateadresseForm(props: Props) {
                         props.onChange({...props.gateadresse, bolignummer: event.target.value})}
                 />
             </InputLinje>
-            <PoststedVelger poststedInformasjon={{postnummer, poststed}} onChange={onPostinformasjonChange(props)} />
+            <PoststedVelger
+                poststedInformasjon={{postnummer, poststed}}
+                onChange={onPostinformasjonChange(props)}
+                feil={getSkjemafeilFraValidering(props.validering ?
```

Comment:
denne må du gjerne forklare for meg :-)

Reply:
Det er bare en hjelpemetode for å sjekke om den finnes et feil-objekt på staten :)

Reply:
Når det er sagt, så lurer jeg litt på å endre signaturen på feil-objektet fra FormValidatoren, slik at man kan passe feilen direkte videre til f.eksinput-felt fra NAV-frontend.